src/main/java/com/cinema/shop/dao/UserDao.java

```diff
@@ -7,4 +7,6 @@
     User add(User user);
 
     Optional<User> findByEmail(String email);
+
+    User getById(Long id);
```

Comment:
```suggestion
    User get(Long id);
```
remember about uniform method names